ARCs/arc-0004.md

```diff
@@ -579,7 +580,8 @@ For any ABI value `x`, we recursively define `enc(x)` to be as follows:
     * `enc(x)` is a single byte whose **most significant bit** is either 1 or 0, if `x` is true or false respectively. All other bits are 0. Note: this means that a value of true will be encoded as `0x80` (`10000000` in binary) and a value of false will be encoded as `0x00`. This is in contrast to most other encoding schemes, where a value of true is encoded as `0x01`.
 
 Other aliased types' encodings are already covered:
-- `string` and `address` are aliases for `byte[]` and `byte[32]` respectively
+- `string` and `bigint` are aliases for `byte[]`
```

Comment:
Two comments about encoding that I think should be in the spec:
* Bigints are encoded big-endian
* Variable-sized bigints don't have a single unique encoding. It's always possible to add more 0s to the beginning and encode a number as a larger byte string. The spec needs to decide if this should be allowed, or if only the most succinct encoding can be used.

Reply:
In 89f3d51 I specified big-endia encoding. I also addressed 0-padding and stated that `enc(x)` SHOULD be as succinct as possible.